user-org-actor/src/main/java/org/sunbird/organization/OrgReadActor.java

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.organization;
+
+import org.sunbird.BaseActor;
+import org.sunbird.OperationValidator;
+import org.sunbird.actor.core.ActorConfig;
+import org.sunbird.actorOperation.OrgActorOperations;
+import org.sunbird.organization.service.IOrgService;
+import org.sunbird.organization.service.OrgServiceImpl;
+import org.sunbird.request.Request;
+
+@ActorConfig(
+  tasks = {"readOrg"},
+  asyncTasks = {}
+)
+public class OrgReadActor extends BaseActor {
+
+  private static IOrgService INSTANCE = new OrgServiceImpl();
```

Comment:
Make INSTANCE to small case, As all upper case letters are used to identify Constants variable.

Reply:
That name looks good to me.